pages/firebase-inputs.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+
+// import { setNewArticle } from '../firebase/api/articles';
+
+const FirebaseArticleInputs = () => {
+  const [isTitleDifferent, setIsTitleDifferent] = useState(false);
+
+  const newArticleHandler = (e) => {
+    e.preventDefault();
+    // const articleObj = { title: e.target[0].value, content: e.target[1].value };
+
+    // eslint-disable-next-line no-console
+    console.log(e.target);
+    // setNewArticle(articleObj);
+  };
+
+  const isTtileDifferentChangeHandler = (event) => {
+    setIsTitleDifferent(!isTitleDifferent);
+  };
+
+  return (
+    <>
+      <form className="m-5" onSubmit={newArticleHandler}>
+        <label className="block" htmlFor="title">
+          Název
+        </label>
+        <input className="mb-4 block border-2" id="title" type="text" />
+
+        <input
+          type="checkbox"
+          id="isTitleDifferent"
+          checked={isTitleDifferent}
+          onChange={isTtileDifferentChangeHandler}
+        />
+        <label className="mb-4" htmlFor="differentTitle">
```

Comment:
I somehow like the `labeledBy` approach more. So the label itself is as stupid as possible.

No strong opinion. Whatever you prefer